src/components/scenes/newAccount/NewAccountReviewScene.tsx

```diff
@@ -89,3 +75,53 @@ const getStyles = cacheStyles((theme: Theme) => ({
     minHeight: theme.rem(3)
   }
 }))
+
+/**
+ * The review scene for new accounts.
+ */
+interface NewAccountReviewProps extends SceneProps<'newAccountReview'> {
+  branding: Branding
+}
+
+export const NewAccountReviewScene = (props: NewAccountReviewProps) => {
+  const { branding, route } = props
+  const dispatch = useDispatch()
+  const { onLogin } = useImports()
+
+  const handleNext = useHandler(() => {
+    logEvent(`Signup_Review_Done`)
+
+    if (onLogin != null) onLogin(route.params.account)
+
+    dispatch(checkAndRequestNotifications(branding)).catch(error =>
+      console.log(error)
+    )
+  })
+
+  return <AccountReviewComponent {...route.params} onNext={handleNext} />
+}
+
+/**
+ * The review scene for upgrading (light) accounts.
+ */
+interface UpgradeReviewProps extends SceneProps<'upgradeAccountReview'> {
+  branding: Branding
+}
+
+export const UpgradeReviewScene = (props: UpgradeReviewProps) => {
+  const { branding, route } = props
+  const dispatch = useDispatch()
+  const { onComplete } = useImports()
+
+  const handleNext = useHandler(() => {
+    logEvent(`Signup_Review_Done`)
+
+    if (onComplete != null) onComplete()
```

Comment:
The logging might need to be updated, since this is a different event. Maybe we should call the event `Upgrade_Review_Done`.

Also, there was an `Airship.clear()` in the old version, that maybe we should put that back?

Reply:
I think the original purpose of Airship.clear() was to dismiss the outermost airship in LoginUiProvider, which is no longer needed